content/sales.rst

```diff
@@ -0,0 +1,152 @@
+Sales and Marketing
+###################
+
+:menu_order: 008
+
+.. contents::
+
+
+Introduction
+============
+
+This page briefly describes operational Sales details.
+
+
+Product Demo
+============
+
+Please see the `DemoSession` wiki page for the demo session structure.
+
+
+Demo text template to use
+-------------------------
+
+The demo session content is similar to the documentation in that it is aimed at
+system administrators and other individuals involved.
+
+Please send your criteria of requirements so that we can
+better answer your questions in advance of the demo.
+
+For those involved in procurement, please contact the Sales team.
+
+**Pre-demo session:**
+
+We recommend obtaining the trial version and installing until the
+`initial stage <https://www.sftpplus.com/documentation/sftpplus/latest/getting-started.html>`_
+
+In that way, you are able to continue after the demo session.
+
+
+Items to consider agreeing to a demo:
+-------------------------------------
+
+Is the customer actively pursuing a trial or considering a trial?
+
+If they are still considering a trial, it would be best to get a list of
+requirements first, and send through response.
+
+
+Is it attended by administrators / individuals involved in managing?
+--------------------------------------------------------------------
+
+Like readership of the documentation page, the session should be attended by
+at least an administrator or individual involved in managing the roll-out.
+
+This will increase the chance of knowledge transfer inside their organization
+during the trial and decrease instances of conducting more than one session.
+
+It will also smooth the process when the customer does the initial
+configuration during the trial.
+
+
+When demo session is agreed ask for the following:
+--------------------------------------------------
+
+* Dates available
+* Times available (and local timezone, so that we know what GoToMeeting
+  toll free / long distance numbers to provide)
+* Number of attendees (max 4 attendees incl 1 organiser and 1 co-organiser in
+  case or max 5 attendees if 1 organiser)
+* Platform
+* Protocols
+* List or criteria of requirements to know what to set up in advance
+
+
+Sending out demo session details
+--------------------------------
+
+* Session is held in the first 10 minutes. After that it may be rescheduled.
+
+* Phone bridge is recommended, if not, use a VOIP headset.
```

Comment:
Maybe this can also be part of the template.
Ask them to join via phone bridge, and if they can't  to join VOIP with dedicated headset and not use the microphone included in the laptops